### PR TITLE
Keyboard shortcut overview

### LINKED ISSUE #310: [Feature] Provide an overview of the available keyboard commands
**Problem**

The currently available keyboard commands are only visible if you take your time, hover over the respective buttons and actionable items, and wait for the tooltip. There are several problems:

- If you do not know which elements can be operated with the keyboard, there is no way of discovering the keyboard shortcuts
- There is no central place where all the keyboard shortcuts are listed as a reference
- People not knowing there are keyboard shortcuts most probably won't find them

**Proposed Solution**

There should be a central place for easy access with all available keyboard shortcuts. Eather as an additional menu item (maybe with a question mark) or as a modal popover.
This could also be the place for a very short help text with the basic information on how to use the tool.

I like the way how it's done in Confluence: 
Screenshot from 2021-05-21 09-55-07

### PR DESCRIPTION
**Pull request by naweidner**
This fixes #436  
This fixes #310 

1. "Play/Pause Video" now in one line
2. Button now sticky to bottom of navbar (navbar is now fixed in position)
3. Navbar is now covering whole page-height (sticky and scrollable, if height ist too small)
4. -
5. Layout is now centered and responsive, content is not overlapping anymore on small widths

Full-Width
full-width

Half-width
half-width

Small-width
small-width

### CODE REVIEW

**Review by lkiesow — CHANGES_REQUESTED** [review]
Looks good. My only complaint is that we are missing a title tag. Can we add that for the keyboard shortcut button?

File: src/main/MainMenu.tsx [hunk 1]
```diff
@@ -34,6 +34,9 @@ const MainMenu: React.FC<{}> = () => {
     flexShrink: 0,
     alignItems: 'center',
     padding: '20px',
+    height: '95vh',
+    position: 'fixed',
```
Comment (Arnei) [inline src/main/MainMenu.tsx:38]: Minor thing, but this causes the right borderline of the main menu to not line up precisely with the bottom of the page anymore.

**Comment by Arnei** [comment]
> Here is a screenshot of my changes. The borderline of the main menu now lines up with the bottom of the page an we have a title tag.
> 
> keyboard-controls

I'm actually not a fan of text on this button, it makes it look to similar to the "real" main menu options. I think it should there should be clear visual distinction that the keyboard button is not directly related to the "main" buttons that pertain to the "main workflow" of the editor.

Ideally the control for accessing keyboard information would not be in the main menu side bar at all, but in a header or something like that. Alas, we don't have that, so this is the next best thing.

**Comment by lkiesow** [comment]
@Arnei, what do you think about my suggestion from above? A title tag identical to the aria label?

File: src/main/MainMenu.tsx (hunk 1 above)
Comment (lkiesow) [reply src/main/MainMenu.tsx]: For me, this seems to work better than before in both Firefox and Chromium.

**Comment by Arnei** [comment]
> @Arnei, what do you think about my suggestion from above? A title tag identical to the aria label?

@lkiesow I ignored that suggestion initially because it really confused me and I only now have realized that you are talking about title attributes. I am absolutely in favor of adding that.

**Comment by naweidner** [comment]
The button now looks like this. Removed the title and made the background (when this menu is active) a bit smaller

Shortcut-Button

**Pull request merged**